### DISCUSSION TITLE
Possessor Mode

### DISCUSSION THREAD

**Discussion by Toy203** [gpstar81/GPStar-proton-pack#308] [Ideas]
Definitely not a priority but i'd love to eventually have a "Possessor" mode for the pack. It could have just blinking red ciclotron lights instead of rotating, and red power cell for Frutto configs, and maybe growling angrly every now and then instead of the regular humming. Smoke too i guess.

Would be fun to have an option to randomly trigger it on X amount of time too. The pack could be left on display during events and randomly get "possessed".

**Comment by nomakewan** [comment]
Closing as this is now under consideration in #338
